CONTRIBUTING.md

```diff
@@ -193,6 +193,28 @@ Reviewers, please check the following items when reviewing a pull-request:
     * Are the tests maintainable?
     * Is the code in the right namespace/class/function?
 
+### Format the sources
+
+Clang-Format may format the sources differently depending on the version used.
+To ensure that the formatting matches the expectations of the CI suite,
+invoke Clang-Format of the correct version from the container (be sure to use the correct image tag):
+
+```
+docker run --rm -v ${PWD}:/repo ghcr.io/opencyphal/toolshed:ts22.4.3 ./build-tools/bin/verify.py build-danger-danger-repo-clang-format-in-place
```

Comment:
This is currently only working if you first go inside docker (`--it`), install nunavut (`pip install nunavut`), and then run `./build-tools/bin/verify.py build-danger-danger-repo-clang-format-in-place` being there. We probably should somehow "teach" the `verify.py` to install this dependency (and after that we can a bit simplify GH actions by removing `pip install nunavut`).